### DISCUSSION TITLE
Raw postfix for where expression

### DISCUSSION THREAD

**Discussion by receiver1** [catfan/Medoo#1098] [Ideas]
I think it’s worth adding a postfix (like r(-aw)) that assumes the absence of any operator in `WHERE` expressions, so that this can be conveniently combined with Medoo::raw. I suggest this because I encountered an `IN` issue in Medoo described in https://github.com/catfan/Medoo/issues/1079. As for me, this is a good alternative and a breath of air for complex and large queries

Just for example (let's do without logic):
```php
$database->select('users', ['id', 'name','age'], [
  'id[r]' => Medoo::raw('IN (SELECT `id` FROM `users` WHERE `age` = 5)'),
]
```
This should generate an SQL query like this:
```sql
SELECT `id`, `name`, `age` FROM `users` WHERE `id` IN (SELECT `id` FROM `users` WHERE `age` = 5)
```
So we don't have an equals operator after `id` in `WHERE`.

**Comment by catfan** [comment]
Thank you for your feedback.

I think I will implement generating subquery in a new way. 🤔
